docs/pin.rst

```diff
@@ -170,3 +176,10 @@ to that.
         whatever is connected to it. Human body has quite a large capacitance,
         so touching the pin gives a dramatic change in reading, which can be
         detected.
+
+
+The default pullup/down states for each type of pin are as follows:
+
+* ``read_analog`` - there is no pull.
+* ``read_digital`` - ``pin.PULL_DOWN``
+* ``is_touched`` - ``pin.PULL_UP``
```

Comment:
It's a bit confusing listing it like this.  First, read_analog/read_digital/is_touched are not pins but rather states of a pin (ie a particular mode).  Second, you can't actually change the pull for any mode except read_digital.

So I'd suggest some wording like this: "The pull mode for a pin is automatically configured when the pin changes to an input mode.  Input modes are when you call read_analog/read_digital/is_touched.  The pull mode for these is, respectively, NO_PULL, PULL_DOWN, PULL_UP.  Only when in read_analog mode can you call set_pull to change the pull mode from the default."